cmake/modules/compiler_config.cmake

```diff
@@ -126,6 +132,12 @@ if (CXX_SUPPORTS_NO_UNKNOWN_PRAGMAS)
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-unknown-pragmas")
 endif()
 
+check_cxx_compiler_flag("-Wno-unused-local-typedefs"
+  CXX_SUPPORTS_NO_UNKNOWN_PRAGMAS)
+if (CXX_SUPPORTS_NO_UNKNOWN_PRAGMAS)
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-unused-local-typedefs")
+endif()
```

Comment:
Same here (`-CXX_SUPPORTS_NO_UNUSED_LOCAL_TYPEDEFS` instead of `CXX_SUPPORTS_NO_UNKNOWN_PRAGMAS`)?